start.php

```diff
@@ -90,6 +90,7 @@ function hypeapps_ui_get_entity_handler(ElggEntity $entity) {
 			'videolist_item' => 'videolist',
 			'album' => 'photos',
 			'image' => 'photos',
+            'poll' => 'poll',
```

Comment:
Indentation is a bit off. If you use PHPStorm, I can send you my settings export.

Reply:
Yes, that would be great. I'm using IntelliJ, so no compatibility issues there.